charter.md

```diff
@@ -1,52 +1,124 @@
 ﻿# Kubernetes Steering Committee Charter
 
-This document outlines the mission, scope, and objectives of the Kubernetes steering committee.
+This document outlines the mission, scope, and objectives of the Kubernetes
+Steering Committee.
 
 ## Mission
 
-The role of the steering committee is to instantiate the formal process
-for Kubernetes governance. In addition to defining the governance
-process, it is important to provide a means for iterating on processes 
-defined by the steering committee.  The role of the steering committee 
-is to be a live, responsive body that can refactor and reform as necessary 
-to adapt to a changing project and community.
+The Kubernetes Steering Committee is the governing body of the Kubernetes
+project, providing decision-making and oversight pertaining to the Kubernetes
+project bylaws, sub-organizations, and financial planning.  The Steering 
+Committee also defines the project values and structure.
 
-## Scope and Objectives
-The Steering Committee has a set of rights and responsibilities including the
-following:
+## How
 
-* Decide which project subgroups are responsible for each area of the project
-  and to delegate appropriate authority those groups.
-* Define, evolve, and defend the vision, values, mission, and scope of the
-  project.
+* Provide top-level accountability for the health of the project.
+* Delegate management<sup>[1](#footnote1)</sup> to other
+  parties<sup>[2](#footnote2)</sup>.
+* Adapt the role and structure of the Steering Committee as needed to meet the
+  needs of the project.
+* Responsibilities not explicitly delegated to other
+  parties<sup>[2](#footnote2)</sup> through their charters reside with
+  the Steering Committee.
+
+## Direct responsibilities of the Steering Committee
+
+The following responsibilities belong to the Steering Committee.
+
+* Act as escalation point of last resort for decisions important to the 
+  project that cannot be resolved within SIGs or Committees.
+* Delegate project responsibilities and ownership to SIGs and Committees.
```

Comment:
We should be explicit that the intent is to delegate decisions on all technical matters to the appropriate SIG